man/tpm2_createak.1.md

```diff
@@ -71,6 +71,12 @@ loaded-key:
 
     Format selection for the signature output file.
 
+  * **-q**, **\--ak-qualified-name**=_FILE_:
```

Comment:
"qualified-name" to keep option name consistent between tools.

Reply:
FYI though, this is consitent with the naming conventions in the tpm2_createak tool. We have -n as ak-name, not just name. So either we go for global consistency, or tool consistency. I am not sure which one is better per se, as neither is ideal in this case, but I would lean towards local consistency.

If we want to correct any difficencies, we can add alias so something like:
-n, --ak-name, --name
-q --qualified-name --ak-qualified-name all work.

But I would leave that for future work.

Reply:
sounds good